<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KWin::DetectWidget</class>
 <widget class="QWidget" name="KWin::DetectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>318</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="textLabel9">
     <property name="text">
      <string>Information About Selected Window</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="Line" name="line1">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>Class:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLabel" name="class_label">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="textLabel3">
     <property name="text">
      <string>Role:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLabel" name="role_label">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="textLabel4">
     <property name="text">
      <string>Type:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLabel" name="type_label">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="textLabel8">
     <property name="text">
      <string>Title:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLabel" name="title_label">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="textLabel13">
     <property name="text">
      <string>Machine:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLabel" name="machine_label">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Match Strategy</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="use_class">
        <property name="whatsThis">
         <string>For selecting all windows belonging to a specific application, selecting only window class should usually work.</string>
        </property>
        <property name="text">
         <string>Use window &amp;class (whole application)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="use_role">
        <property name="whatsThis">
         <string>For selecting a specific window in an application, both window class and window role should be selected. Window class will determine the application, and window role the specific window in the application; many applications do not provide useful window roles though.</string>
        </property>
        <property name="text">
         <string>Use window class and window &amp;role (specific window)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="use_whole_class">
        <property name="whatsThis">
         <string>With some (non-KDE) applications whole window class can be sufficient for selecting a specific window in an application, as they set whole window class to contain both application and window role.</string>
        </property>
        <property name="text">
         <string>Use &amp;whole window class (specific window)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="match_title">
        <property name="text">
         <string>Match also window &amp;title</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
